scripts/aws_helper.py

```diff
@@ -49,6 +52,10 @@ def get_credentials(bucket_name: str):
                 assumed_role_object["Credentials"]["SecretAccessKey"],
                 assumed_role_object["Credentials"]["SessionToken"],
             )
+        if set_credentials:
```

Comment:
We should never overwrite the system environment variables.

You should take these credentials and add them into the environment of the command

Reply:
Yes, I was concerned by removing the default credentials by doing this.